projects/media-viewer/src/lib/toolbar/icp-toolbar/icp-toolbar.component.html

```diff
@@ -35,4 +35,14 @@
       </button>
     </div>
   </div>
+  <div id="toolbarViewerRight">
+    <div class="splitToolbarButton">
+      <button class="toolbarButton"
+              id="icp-participants-list" tabindex="0" title="Participants list"
+              (click)="showParticipantsList()">
+        <span *ngIf="!(toolbarEventService.icp.icpParticipantsListVisible | async)" data-l10n-id="icp-stop-presenting_label">Show Participants</span>
```

Comment:
only thing that's different here is the text, so we should probably use the if statement on that
{{ 'Hide Participants' ? (toolbarEventService.icp.icpParticipantsListVisible | async) : 'Show participants'  }}